internal/reports/create.go

```diff
@@ -0,0 +1,50 @@
+package reports
+
+import (
+	"bytes"
+	"fmt"
+	"strings"
+)
+
+type CreateHandOff struct {
+	AwsAccountId      string
+	AwsHostedZoneName string
+	AwsRegion         string
+	ClusterName       string
+	ArgoCDUrl         string
+	ArgoCDUsername    string
+	ArgoCDPassword    string
+	VaultUrl          string
+	VaultToken        string
+}
+
+func BuildCreateHandOffReport(clusterData CreateHandOff) bytes.Buffer {
+
+	var handOffData bytes.Buffer
+	handOffData.WriteString(strings.Repeat("-", 70))
+	handOffData.WriteString(fmt.Sprintf("\nCluster %q is up and running!:\n", clusterData.ClusterName))
+	handOffData.WriteString(strings.Repeat("-", 70))
+
+	handOffData.WriteString("\n\n--- AWS ")
+	handOffData.WriteString(strings.Repeat("-", 62))
+	handOffData.WriteString(fmt.Sprintf("\n AWS Account Id: %s\n", clusterData.AwsAccountId))
+	handOffData.WriteString(fmt.Sprintf(" AWS hosted zone name: %s\n", clusterData.AwsHostedZoneName))
+	handOffData.WriteString(fmt.Sprintf(" AWS region: %s\n", clusterData.AwsRegion))
+	handOffData.WriteString(strings.Repeat("-", 70))
+
+	handOffData.WriteString("\n\n--- ArgoCD ")
+	handOffData.WriteString(strings.Repeat("-", 59))
+	handOffData.WriteString(fmt.Sprintf("\n URL: %s\n", clusterData.ArgoCDUrl))
+	handOffData.WriteString(fmt.Sprintf(" username: %s\n", clusterData.ArgoCDUsername))
+	handOffData.WriteString(fmt.Sprintf(" password: %s\n", clusterData.ArgoCDPassword))
+	handOffData.WriteString(strings.Repeat("-", 70))
+
+	handOffData.WriteString("\n\n--- Vault ")
+	handOffData.WriteString(strings.Repeat("-", 59))
+	handOffData.WriteString(fmt.Sprintf("\n URL: %s\n", clusterData.VaultUrl))
+	handOffData.WriteString(fmt.Sprintf(" token: %s\n", clusterData.VaultToken))
+	handOffData.WriteString(strings.Repeat("-", 70))
```

Comment:
Should we clean the terminal window before closing it, to avoid passwords to live on the terminal buffer for too long. In case of live-demos, this may make some devs to want to suppress this screen to avoid conference attacks by people watching the live stream or something.  ;)

Reply:
I agree, but I would like to propose a different approach:

idea a) ask the user like "hey, we're going to provide some sensitive data, do you want to see it now? (yes/no)

idea b) tell the user that the sensitive data is stored at the place `.../folder/file` and he can use a decryption key to open it

the current solution is something quick for us to start, we can keep improving it.